testfiles/ASHRAE9012016_OutPatientHealthCare_Denver.idf

```diff
@@ -48330,13 +48364,13 @@
     AHU-1_Fan-AHU-1 HumidifierNode,  !- Fan Outlet Node Name
     AHU-1_ExtraElecHeatC-AHU-1_ExtraWaterHeatCNode;  !- Setpoint Node or NodeList Name
 
-  SetpointManager:MixedAir,
-    AHU-1_ExtraWaterHeatC Air Temp Manager,  !- Name
-    Temperature,             !- Control Variable
-    AHU-1 Supply Equipment Outlet Node,  !- Reference Setpoint Node Name
-    AHU-1_HeatC-AHU-1_FanNode,  !- Fan Inlet Node Name
-    AHU-1_Fan-AHU-1 HumidifierNode,  !- Fan Outlet Node Name
-    AHU-1 Supply Equipment Outlet Node;  !- Setpoint Node or NodeList Name
+!  SetpointManager:MixedAir,
+!    AHU-1_ExtraWaterHeatC Air Temp Manager,  !- Name
+!    Temperature,             !- Control Variable
+!    AHU-1 Supply Equipment Outlet Node,  !- Reference Setpoint Node Name
+!    AHU-1_HeatC-AHU-1_FanNode,  !- Fan Inlet Node Name
+!    AHU-1_Fan-AHU-1 HumidifierNode,  !- Fan Outlet Node Name
+!    AHU-1 Supply Equipment Outlet Node;  !- Setpoint Node or NodeList Name
```

Comment:
Is this kept in here for any specific reason?  If not, maybe we should just take it out.

Reply:
No reason to keep it. Removed.